auth-lib/src/main/java/com/spotify/sdk/android/auth/app/Sha1HashUtil.java

```diff
@@ -0,0 +1,36 @@
+package com.spotify.sdk.android.auth.app;
+
+import java.io.UnsupportedEncodingException;
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+
+class Sha1HashUtil {
```

Comment:
This class could be final, and all methods and fields static. This is a helper/internal detail implementation.

Reply:
Made it final, kept the methods as non static to be able to test it easily as I don't see much drawbacks.